setup.py

```diff
@@ -277,6 +277,13 @@ def run(self):
 cwd = os.path.dirname(os.path.abspath(__file__))
 lib_path = os.path.join(cwd, "torch", "lib")
 
+# Check if you remembered to check out submodules
+gloo_cmake = os.path.join(lib_path, "gloo", "CMakeLists.txt")
+if not os.path.exists(gloo_cmake):
+    print "Could not find {}".format(gloo_cmake)
+    print "Did you run 'git submodule update --init --recursive'?"
```

Comment:
This breaks in Py3

Reply:
You DON'T want recursive, it will pull in gtest and gloo's nccl, you don't want that.